README.rst

```diff
@@ -1,3 +1,30 @@
+Magic Wormhole progress
+=======================
+Magic Wormhole implementation now works with both Python 2 and Python 3.
+
+It is required the latest git version of Magic Wormhole.
```

Comment:
Absolutely, this was a temporary edit I did only to show the state of the implementation.

Also regarding Magic Wormhole, there was a recent new stable release that we can use  (0.10.2 of 20 days ago).
I successfully tested it, so now we don't need anymore the git version.